modules/platform/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/BatchUploadFixture.java

```diff
@@ -132,34 +128,25 @@ private void itCanUseBatchUpload(boolean noDrop) throws IOException {
             assertEquals(batchId, node.get("batchId").asText());
             assertEquals("0", node.get("fileIdx").asText());
             assertEquals("normal", node.get("uploadType").asText());
-            // TODO NXP-18247 when the actual uploaded size is returned
-            // assertEquals(fileSize1, node.get("uploadedSize").asText());
```

Comment:
I think we can keep this comment.
It's interesting to know that the uploaded size could be checked if implemented.

Reply:
I got it wrong and thought `uploadedSize` was only filled when `isMultipart` was true.